CCS (docs) review of dashboard content

laubai opened the pull request:
Many minor and moderate changes to resolve:

- en-US language errors
- style problems (to improve clarity, readability)
- formatting errors (of things that customers need to copy and paste)
- incorrect steps (based on testing in current UI)
- incomplete or missing information


In particular, please review the TODO item in data/quickstarts/connect-rhosak-notebook-quickstart.yaml (adding a prerequisite) because I'm not sure if this is accurate, or if this tutorial might only be visible if Kafka is installed.

Please also review the Seldon tutorials, in particular any line with "Run a single prediction" because these were the sentences that were hardest to parse and I'm not 100% certain of their correctness.

jeff-phillips-18 commented:
@laubai Do you want to tag specific reviewers who might best be able to verify the textual changes?

laubai commented:
@cfchase @sophwats You're probably the best folks to review these changes.

anishasthana reviewed (changes requested):
@laubai can you please squash your commits?

laubai commented:
@anishasthana Done, sorry about the wait.

laubai commented:
I squashed the commits and there were conflicts so I rebased and now my commits are un-squashed and i'm having trouble re-squashing them. Any suggestions?

jeff-phillips-18 commented:
> I squashed the commits and there were conflicts so I rebased and now my commits are un-squashed and i'm having trouble re-squashing them. Any suggestions?

Looks good now.

anishasthana reviewed (commented):
At a high level the changes look good to me, but I think we'll still want sign-off from someone like @cfchase  or @taneem-ibrahim

anishasthana reviewed (approved):
"approving" to get rid of the "requested changes" flag

cfchase commented on data/quickstarts/deploy-python-model-quickstart.yaml:
```diff
@@ -8,69 +8,75 @@ spec:
       description: |-
-        ### To create a new repository from the 2si Flask template:
+        ### To create a new repository from the s2i Flask template:
         1. Navigate to `https://github.com/cfchase/s2i-flask-notebook`.
```
This repo has been transferred to https://github.com/opendatahub-io/odh-s2i-project-simple

laubai replied:
Thanks! Will update today.

cfchase reviewed (changes requested):
One URL change, otherwise it looks great.  The Marketplace comment is food for thought, but not necessary for merge.

laubai commented on data/getting-started/seldon-deploy.md:
```diff
@@ -14,17 +14,17 @@ Features include:
 Documentation on the Seldon Deploy Red Hat marketplace install is at [https://deploy-master.seldon.io/docs/getting-started/redhat-installation/](https://deploy-master.seldon.io/docs/getting-started/redhat-installation/)
 
-### Subscribe to the operator on Market place 
+### Subscribe to the operator on Market place
```
Agreed, will update today.
